### PR TITLE
feat(func): Make juju channel optional for functional test

### PR DESCRIPTION
**Pull request by jneo8**

### CODE REVIEW

**Review by sudeephb — APPROVED** [review]

**Review by Pjack — APPROVED** [review]

File: .github/workflows/_func.yaml [hunk 1]
```diff
@@ -52,6 +52,10 @@ on:
         required: false
         type: boolean
         default: false
+      juju-channel:
```
Comment (rgildein) [inline .github/workflows/_func.yaml:55]: You did not used this anywhere. It's need to be passed to [actions-operator](https://github.com/charmed-kubernetes/actions-operator/blob/main/action.yaml#L47).

**Review by rgildein — CHANGES_REQUESTED** [review]
You did not apply juju-channel anywhere.

File: .github/workflows/_func.yaml (hunk 1 above)
Comment (jneo8) [reply .github/workflows/_func.yaml:55]: It need to define here so I can pass `juju-channel` through?

I can't pass to `actions-operator` if we don't define it.

Comment (rgildein) [reply .github/workflows/_func.yaml:55]: Yes, you did it right, but you need to use it in step with actions/operator.

Comment (rgildein) [reply .github/workflows/_func.yaml:55]: add single line (nol 85):
```yaml
        with:
          provider: ${{ inputs.provider }}
          juju-channel: ${{ inputs.juju-channel }}
```

**Review by rgildein — APPROVED** [review]
Thanks

**Review by agileshaw — APPROVED** [review]

**Pull request merged**